### PR TITLE
Switch to a Glimesh janus-gateway mirror

### PR DESCRIPTION
**Pull request by clone1018**
The mirror is intended to reduce the checkout time and size required for any usage of this repo. With these changes the checkout of the glimesh.tv repo can now happen purely unauthenticated.

The other side of this PR is the new repo @ https://github.com/Glimesh/janus-gateway-mirror which does not maintain any state, and instead will mirror whatever janus-gateway code we need to publish to npm at https://www.npmjs.com/package/janus-gateway-mirror

### CODE REVIEW

File: src/FtlPlayer.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { Janus, PluginHandle, Message, JSEP } from "janus-gateway";
+import { Janus, PluginHandle, Message, JSEP } from "janus-gateway-mirror";
```
Comment (clone1018) [inline src/FtlPlayer.ts:1]: Ideally these would stay the same, but I could not find any sane way of doing it with light googling.

File: webpack.config.js [hunk 2]
```diff
@@ -8,7 +8,7 @@ module.exports = {
         // instead it expects a global object called 'adapter' for that.
         // Let's make that object available.
         new webpack.ProvidePlugin({
-            adapter: 'webrtc-adapter'
+            adapter: ['webrtc-adapter', 'default']
```
Comment (clone1018) [inline webpack.config.js:11]: Not 100% sure why this change was needed, but it looks like the slight change in dependency tree moved this inside a module and webpack recommends this syntax to go inside of it.

**Review by haydenmc — APPROVED** [review]
I have to imagine that eventually we'll just part ways with `janus.js` and wind up writing our own, in which case this will all become easier :D
